README.md

```diff
@@ -48,9 +53,12 @@ Suppose you had the following form in your HTML file. Use jQuery to write a sing
     <input type="submit" value="Submit" id="submit">
   </form>
 ```
+```
+    var user_input = $("#color").append('<li>' + color + '</li>');
```

Comment:
Close. You are selecting the right element, the text box with id of color, but you don't need to append it anywhere. Instead, you need to use the `.val()` method to get what the user typed in. So that would be `var user_input = $('#color').val()`.